builder.toml

```diff
@@ -4,7 +4,7 @@ build-image = "heroku/pack:18-build"
 run-image = "heroku/pack:18"
 
 [lifecycle]
-version = "0.8.0"
```

Comment:
Should we do this in `functions-builder.toml` too?

Reply:
I thought about it..but then remembered you had an open item to merge the two builders into one. So I was thinking of ignoring the other builder for now.